src/api/app/assets/stylesheets/webui2/webui2.css.scss

```diff
@@ -25,3 +25,7 @@
 @import 'icons';
 @import 'kaminari';
 @import 'system-status';
+
+.no-wrap {
```

Comment:
There is the [`.text-nowrap` Bootstrap class](https://getbootstrap.com/docs/4.1/utilities/text/#text-wrapping-and-overflow) achieving the same.